### PR TITLE
[RISC-V][JIT] Fix assert on passing double in struct

### PR DESCRIPTION
**Pull request by t-mustafin**
cc @jakobbotsch @gbalykov @clamp03 @alpencolt

### CODE REVIEW

**Comment by clamp03** [comment]
You need to run jit-format. https://github.com/dotnet/jitutils/blob/main/doc/formatting.md

**Review by jakobbotsch — APPROVED** [review]

**Comment by jakobbotsch** [comment]
The failure is stale, the job was renamed in #80718

**Review by gbalykov — COMMENTED** [review]
LGTM

File: src/coreclr/jit/scopeinfo.cpp [hunk 1]
```diff
@@ -961,19 +961,23 @@ void CodeGen::psiBegProlog()
                 else
                 {
                     regType = compiler->mangleVarArgsType(lclVarDsc->TypeGet());
+#ifdef TARGET_LOONGARCH
```
[the remark this answers is not in the document]
Comment (t-mustafin) [reply src/coreclr/jit/scopeinfo.cpp:964]: @shushanhf My mistake, sorry for that.

**Pull request merged**